addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -761,6 +761,7 @@ var StatementModel = BasicModel.extend({
                 args: [ids, values],
                 context: self.context,
             })
+            .then(this._validatePostProcess.bind(this))
```

Comment:
can't you divide this function in another way where it wouldn't need to create an empty hook function?

Reply:
I can't think of a way that would be sensibly better.